types/block_test.go

```diff
@@ -438,6 +484,15 @@ func TestBlockMaxDataBytes(t *testing.T) {
 	}
 }
 
+func TestEmptyEvidenceBytes(t *testing.T) {
+	ed := tmproto.EvidenceData{}
+	t.Log(ed.Size())
+}
+
+func TestMaxDataBytesNoEvidence(t *testing.T) {
+	t.Log(MaxDataBytesNoEvidence(1000, 1))
+}
+
```

Comment:
These seem incomplete.

Reply:
yeah I think they are covered by other tests so can be removed